centreon/src/Core/Media/Infrastructure/Repository/FileWriteMediaRepository.php

```diff
@@ -53,6 +53,24 @@ public function add(NewMedia $media): int
         return $status;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function update(Media $media): void
+    {
+        if ($media->getData() === null) {
```

Comment:
null or empty no ? because data is a string nullable.

Reply:
Fixed